jwst/datamodels/tests/test_models.py

```diff
@@ -377,24 +123,19 @@ def test_multislit():
         assert slit.barshadow.shape == (0, 0)
 
 
-def test_secondary_shapes():
-    with ImageModel((256, 256)) as dm:
-        assert dm.dq.shape == (256, 256)
-        dm.dq = [1]
-
-
 def test_multislit_from_image():
     with ImageModel((64, 64)) as im:
         with MultiSlitModel(im) as ms:
             assert len(ms.slits) == 1
             assert ms.slits[0].data.shape == (64, 64)
 
 
-def test_multislit_from_fits_image():
+def test_multislit_from_saved_image(tmp_path):
```

Comment:
These could go in a multislit-specific test module as well

Reply:
Done.